### PR TITLE
updated annotations for idr0023

### PR DESCRIPTION
**Pull request by eleanorwilliams**

### CODE REVIEW

**Comment by eleanorwilliams** [comment]
These annotations are now in idr-demo2 so safe to merge.

File: idr0023-szymborska-nuclearpore/idr0023-study.txt [hunk 1]
```diff
@@ -1,91 +1,85 @@
...
...al event was calculated as the number of photons collected after background subtraction, divided by the camera calibration factor determined by the manufacturer. To exclude events with low localization precision, those with an integrated intensity of less than 500 photons were filtered out.  The lateral drift in the images was corrected using an adapted correlation-based method (32, 33). Briefly, the localization data was split into equal segments of several thousand frames and an SR image was reconstructed from each segment. A correlation map between adjacent segments was calculated using the Kuglin-Hines algorithm (34). The position of the peak in the correlation map was determined by a 2D Gaussian fit and the relative shift was calculated as a difference between peak positions in subsequent maps. The final drift curves were generated by linear interpolation and then subtracted from the localization data. The procedure was iterated until no residual drift was observed. The final SR images were reconstructed from the intensity-filtered and drift- corrected sets of localizations using 10 nm pixelation. Each localization event was rendered as a single gray value of a 16-bit image. "
```
Comment (joshmoore) [inline idr0023-szymborska-nuclearpore/idr0023-study.txt]: Unicode issues in this long line.

Comment (eleanorwilliams) [inline idr0023-szymborska-nuclearpore/idr0023-study.txt]: Fixed unicode issues.

**Comment by joshmoore** [comment]
Thanks. Merging.

**Pull request merged**